### PR TITLE
StringLen - Add Description and Examples

### PR DESCRIPTION
**Pull request by Xcreen**
Add Description and Examples

### CODE REVIEW

**Comment by shaedrich** [comment]
> We like one liner code examples, they can be tested using the unit tests if a result is given. Just omit `cfset` (https://cfdocs.org/utilities/json/)

Just write it right into `writeOutput`.

File: data/en/stringlen.json [hunk 1]
```diff
@@ -5,13 +5,28 @@
...
 	],
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/stringlen.html"}
 	},
-	"examples": [],
+	"examples": [
+		{
+			"title":"StringLen() example",
+			"description":"Output the length of the string \"Hello World\".",
+			"code":"strLength = StringLen(\"Hello World\");\nWriteOutput(strLength);",
```
Comment (shaedrich) [inline data/en/stringlen.json]: > We like one liner code examples, they can be tested using the unit tests if a result is given. Just omit `cfset` (https://cfdocs.org/utilities/json/)

Just write it right into `writeOutput`.

File: data/en/stringlen.json [hunk 2]
```diff
@@ -5,13 +5,28 @@
...
+			"result":"11",
+			"runnable":true
+		},
+		{
+			"title":"StringLen() using as a member function",
+			"description":"Output the length of the string \"Hello World\".",
+			"code":"helloWorldString = \"Hello World\";\nWriteOutput(helloWorldString.len());",
```
Comment (shaedrich) [inline data/en/stringlen.json]: same suggestion as above.

**Comment by Xcreen** [comment]
@shaedrich Can you review again? Its also failing now due:
> Syntax Error, invalid Expression ["Hello World".len()]

But the example its running fine under Lucee 4.5 / 5

**Comment by shaedrich** [comment]
As far as I can see as well as you this should work. I wonder, too...

**Comment by pfreitag** [comment]
For some reason the `evaluate` function doesn't like it...

The one liner examples are run through evaluate to test the result, I just tried this in commandbox REPL as well, and throws the same error:

```
CFSCRIPT-REPL: evaluate("""Hello World"".len()");
ERROR
Syntax Error, invalid Expression ["Hello World".len()]
```

Not sure how to work around this, since it is an issue with evaluate on Lucee.

**Comment by pfreitag** [comment]
This works `toString('Hello World').len()` not as elegant though

**Comment by shaedrich** [comment]
Couldn't you try to catch this inside the build job, @pfreitag?

**Comment by pfreitag** [comment]
yeah, good call - lets do that I'll work on a fix

**Comment by pfreitag** [comment]
Thanks @Xcreen I updated the test runner to ignore that so it passes now.

**Comment by Xcreen** [comment]
Perfect :)

**Comment by shaedrich** [comment]
Thanks @pfreitag :) :+1:

**Pull request merged**